bin/node/src/aleph_cli.rs

```diff
@@ -3,9 +3,8 @@ use std::path::PathBuf;
 use aleph_primitives::DEFAULT_UNIT_CREATION_DELAY;
 use clap::{ArgGroup, Parser};
 use finality_aleph::UnitCreationDelay;
-use sc_service::Configuration;
 
-const DEFAULT_BACKUP_FOLDER: &str = "backup-stash";
+pub const DEFAULT_BACKUP_FOLDER: &str = "backup-stash";
```

Comment:
Now defining this here doesn't make much sense. `chain_spec.rs` seems like a good place, there are many other constants defined there

Reply:
done